core/templates/dev/head/pages/exploration_editor/translation_tab/AudioTranslationBarDirective.js

```diff
@@ -94,6 +94,12 @@ oppia.directive('audioTranslationBar', [
           $scope.audioTimerIsShown = true;
           $scope.audioIsCurrentlyBeingSaved = false;
 
+          document.body.onkeyup = function(e) {
```

Comment:
Try playing with this functionality after switching exploration tabs. I think switching tabs will lead to have multiple events for this button.

Reply:
Checked